test/view_model_tests/main_screen_view_model_test.dart

```diff
@@ -48,6 +48,31 @@ class MockLocalMainScreenViewModel extends MainScreenViewModel {
   }
 }
 
+typedef FunctionType = void Function(MainScreenViewModel model2);
+Widget getMockApp(FunctionType onTap) {
```

Comment:
1. Add a new line here.
2. Rename the function to be consistent with the naming used in other tests.